modules/tinymce/src/core/test/ts/browser/html/SaxParserTest.ts

```diff
@@ -793,6 +794,68 @@ UnitTest.asynctest('browser.tinymce.core.html.SaxParserTest', function (success,
     );
   });
 
+  suite.test('Parse svg urls (default)', function () {
+    const counter = createCounter(writer);
+    counter.validate = false;
+    const parser = SaxParser(counter, schema);
+    writer.reset();
+    parser.parse(
+      '<iframe src="data:image/svg+xml;base64,x"></iframe>' +
+      '<a href="data:image/svg+xml;base64,x">1</a>' +
+      '<object data="data:image/svg+xml;base64,x"></object>' +
+      '<img src="data:image/svg+xml;base64,x">' +
+      '<video poster="data:image/svg+xml;base64,x"></video>'
+    );
+    LegacyUnit.equal(
+      writer.getContent(),
+      '<iframe></iframe><a>1</a><object></object>' +
```

Comment:
Can we put the `a` and `object` on separate lines so when reading the test it's clear that the data has been stripped, but the elements remain?

Reply:
I did it this way, as that's how the rest of the file works. That being said, I agree it'd be better, so I've reworked the URL checks to use a new line per element.